lib/langchain/utils/token_length/google_palm_validator.rb

```diff
@@ -37,6 +37,9 @@ class GooglePalmValidator < BaseValidator
         #
         def self.token_length(text, model_name = "chat-bison-001", options)
           response = options[:llm].client.count_message_tokens(model: model_name, prompt: text)
+
+          raise Langchain::LLM::ApiError.new(response["error"]["message"]) unless response["error"].nil?
```

Comment:
Do you want to extend this out to [here](https://github.com/andreibondarev/langchainrb/blob/main/lib/langchain/llm/openai.rb#L128) as well?